src/Microsoft.DotNet.Interactive.HttpRequestParser/HttpRequestNode.cs

```diff
@@ -109,9 +136,22 @@ public HttpBindingResult<HttpRequestMessage> TryGetHttpRequestMessage(HttpBindin
         {
```

Comment:
Why is it that the `HttpBodyNode` contains a `TryGetBody` that returns a body object but `HttpHeaderNode` does not contain a `TryGetHeader` that returns the header object? Should we create `HttpHeaderNode.TryGetHeader` and move the code for creating headers in there?

Reply:
There's no common type for headers and no uniform API on `HttpRequestMessage` to set them, so I'm not sure adding an abstraction would be helpful.

Reply:
Ah I see - I assumed there is always a common base type with multiple specializations. Sigh.. I wish the `HttpRequestMessage` API had a simpler API to deal with headers.